projects/igniteui-angular/src/lib/core/styles/components/chips/_chips-theme.scss

```diff
@@ -0,0 +1,245 @@
+
+        text-color: $text-color,
+        text-hover-color: $text-hover-color,
+        order-icon-hover-color: $order-icon-hover-color,
+
+        order-icon-color: $order-icon-color,
+        dir-icon-color: $dir-icon-color,
+        remove-icon-color: $remove-icon-color,
+        remove-icon-hover-color: $remove-icon-hover-color,
+    ));
+}
+
+/// @param {Map} $theme - The theme used to style the component.
+/// @requires {mixin} igx-root-css-vars
+/// @requires rem
+/// @requires var
+@mixin igx-chips($theme) {
+    @include igx-root-css-vars($theme);
+
+    %igx-chips-area,
+    %igx-chips-area__item,
+    %igx-chips-area__label,
+    %igx-chips-area__chip-connector {
+        display: flex;
+        align-items: center;
+    }
+
+    %igx-chips-area {
+        justify-content: flex-start;
+        width: 100%;
+    }
+
+    %igx-chips-area__item {
+        user-select: none;
+        cursor: pointer;
+        border-radius: rem(17px);
+        padding: rem(4) rem(16);
+        height: rem(34px);
+        transition: all 120ms ease-in-out;
+        background: --var($theme, 'chip-background');
+
+        &:hover,
+        &:focus{
+            background: --var($theme, 'chip-hover-background');
+
+            %igx-chips-area__text {
+                color: --var($theme, 'text-hover-color');
+            }
+
+            transition: all 120ms ease-in-out;
+
+            .igx-icon {
+                transition: all 120ms ease-in-out;
+
+                &%igx-chips-area__order-icon {
+                    color: --var($theme, 'order-icon-hover-color')
+                }
+            }
+        }
+
+        .igx-icon,
+        .igx-button--icon {
+            width: rem(16px);
+            height: rem(16px);
+            font-size: rem(16px);
+        }
+    }
+
+    %igx-chips-area__text {
+        color: --var($theme, 'text-color');
+        font-size: rem(12px);
+        margin-right: rem(16px);
+    }
+
+    .igx-icon {
+        &%igx-chips-area__dir-icon {
+            width: rem(16px);
+            height: rem(16px);
+            font-size: rem(16px);
+            margin: 0 rem(8px);
+            color: --var($theme, 'dir-icon-color');
+        }
+
+        &%igx-chips-area__order-icon {
+            color: --var($theme, 'order-icon-color');
+            margin-right: rem(4px);
+        }
+
+        &%igx-chips-area__remove-icon {
+            color: --var($theme, 'remove-icon-color');
+            transition: color 120ms ease-in-out;
+
+            &:hover {
+                color: --var($theme, 'remove-icon-hover-color');
+                opacity: 1;
+            }
+        }
+    }
+
+    %igx-chips-area__chip-connector {
+        user-select: none;
+    }
+
+    %igx-chips-area__item--disabled {
+        %igx-chips-area__item {
+            cursor: default;
+            pointer-events: none;
+        }
+
+        %igx-chips-area__label {
+            opacity: .5;
+        }
+    }
+}
```

Comment:
Absolutely hate the idea of coupling the area of where a chip is placed with the theme for a single chip instance.

Reply:
No hate is necessary here :)